### PR TITLE
feat: add development instructions

### PR DESCRIPTION
**Pull request by realaravinth**
For rendered page, please see: https://github.com/realaravinth/interface/blob/docs-hacking/docs/HACKING.md

### CODE REVIEW

File: docs/HACKING.md [hunk 1]
```diff
@@ -0,0 +1,116 @@
...
+
+Kindly delete virtual environment(found at the root of the repository in
+directory named `venv`) and run `make env` to recreate the environment.
+
+### 3.2 I want to discard my current database and create a new one
+
+Delete `instance` directory and run `make migrate`.
+
+```bash
+rm -rf ./instalnce && make migrate
```
Comment (dat-adi) [inline docs/HACKING.md:103]: Could you please convert `instalnce` to `instance`?

**Review by dat-adi — CHANGES_REQUESTED** [review]
Minor spelling change requested, everything else looks good to go.

**Comment by realaravinth** [comment]
Thanks for the quick review, suggested changes are applied

File: docs/HACKING.md [hunk 2]
```diff
@@ -0,0 +1,116 @@
...
+
+Kindly delete virtual environment(found at the root of the repository in
+directory named `venv`) and run `make env` to recreate the environment.
+
+### 3.2 I want to discard my current database and create a new one
+
+Delete `instance` directory and run `make migrate`.
+
+```bash
+rm -rf ./isntance && make migrate
```
Comment (dat-adi) [inline docs/HACKING.md:103]: Uh, `isntance` to `instance`.

Comment (realaravinth) [reply docs/HACKING.md]: Apologies, I apparently can't type today! I think the latest commit should fix it

**Review by dat-adi — APPROVED** [review]
Typo has been fixed, approved for a merge.

**Pull request merged**